src/pages/WriteProductsPage.tsx

```diff
@@ -1,9 +1,67 @@
 import type React from 'react';
+import { useForm, SubmitHandler } from 'react-hook-form';
+
+type FormData = {
+	title: string;
+	url: string;
+	comment: string;
+};
 
 export const WriteProductsPage: React.FC = () => {
+	const { register, handleSubmit, formState: { errors } } = useForm<FormData>();
+
+	const onSubmit: SubmitHandler<FormData> = (data) => {
+		console.log("Submitted Data:", data);
+		alert(`Title: ${data.title}\nURL: ${data.url}\nComment: ${data.comment}`);
```

Comment:
```suggestion
		alert(`投稿完了しました！`);
```
文章任せます。ユーザー側に伝わるようにお願い🙏